src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/categories.yml

```diff
@@ -0,0 +1,67 @@
+admin_category_listing:
+    path: /
+    methods: [GET, POST]
+    defaults:
+        _controller: PrestaShopBundle:Admin\Sell\Catalog\Category:index
+        _legacy_controller: AdminCategories
+
+admin_category_listing_search:
+    path: /search
+    methods: [POST]
+    defaults:
+        _controller: PrestaShopBundle:Admin\Common:searchGrid
+        gridDefinitionFactoryService: prestashop.core.grid.definition.factory.category
+        redirectRoute: admin_category_listing
+        redirectQueryParamsToKeep:
+            - 'id_category'
+
+admin_category_process_bulk_status_enable:
+    path: /bulk-status-enable
```

Comment:
Why having to route for bulk-status, when it's just an action with a boolean to enable or not :/